t/mojo/log.t

```diff
@@ -229,4 +229,27 @@ subtest 'Context' => sub {
   like $buffer,   qr/\[.*\] \[fatal\] \[123\] Mojolicious rocks\n/, 'right fatal message';
 };
 
+subtest 'wrappers' => sub {
```

Comment:
The convention seems to be capitalisation for subtest names, e.g. `subtest 'History' => sub {`, unless the subtest name is a function name, in which case it is quoted, e.g. `subtest '"error"' => sub {`.

As this tests specifically your new `sprintf` wrappers, in my opinion it should be something like `Text formatting wrappers` for the subtest name.